dotcom-rendering/src/components/SignInGate/signInGate.ts

```diff
@@ -15,16 +16,19 @@ import type { SignInGateTestMap } from './types';
 export const signInGateTests: ReadonlyArray<ABTest> = [
 	signInGateMainVariant,
 	signInGateMainControl,
+	signInGateTimesOfDay,
 ];
 
 export const signInGateTestVariantToGateMapping: SignInGateTestMap = {
 	'main-control-5': gateMainControl,
 	'main-variant-5': gateMainVariant,
+	morning: gateMainVariant,
 };
 
 // Component Id does not need to match gate test name, as ab test info passed separately to ophan
 // Consider Id name relevant to the gate component or design. Use snake_case
 export const signInGateTestIdToComponentId: { [key: string]: string } = {
 	SignInGateMainVariant: 'main_variant_5',
 	SignInGateMainControl: 'main_control_5',
+	SignInGateTimesOfDay: 'time_of_day_morning',
```

Comment:
not a blocker but as this relates to the component ID it can just stay as 'main_control_5' as we are not changing the design?